src/items/equipment/soulfire.json

```diff
@@ -29,7 +36,13 @@
       "sturdy": false
     },
     "attuned": false,
-    "bulk": "L",
+    "bulk": "",
```

Comment:
While Fusions do have no bulk, I'm not a fan of just leaving it blank, just leave it as `-`